vividus/src/main/resources/templates/udiff.ftl

```diff
@@ -42,7 +42,7 @@
             renderNothingWhenEmpty: false,
             rawTemplates: { "generic-file-path": fileLine, "tag-file-changed": "<span/>"},
         };
-        let diff2htmlUi = new Diff2HtmlUI(targetElement, `${udiff?no_esc}`, configuration);
+        let diff2htmlUi = new Diff2HtmlUI(targetElement, `${udiff?replace("${", "\\${")?replace("`", "\\`")?no_esc}`, configuration);
```

Comment:
> Why do we do the escaping in the template and not in the java code?

1. it's hard to track the correlation: escaping is required because of usage of js `...`, if this logic is changed, it's hard to recall that java code should be updated as well
2. escaping is not need if string length is less than threshold

> In the second case we can at least test the logic

true, but having straightforward logic it's still needed to test the template work manually